packages/editor-ui/src/components/NodeCreator/NodeItem.vue

```diff
@@ -83,6 +95,12 @@ export default Vue.extend({
 				left: `${this.draggablePosition.x}px`,
 			};
 		},
+		isCommunityNode(): boolean {
+			return this.$store.getters['communityNodes/isCommunityNode'](this.nodeType.name);
+		},
```

Comment:
would be nice if we could just add a flag somewhere to this object as we add them to the store...